### ISSUE TITLE
GQLV2: Public mutation to create events

### ISSUE THREAD

**Issue by npettiaux** [opencollective/opencollective#3775]
Dear,
I have seen in https://docs.opencollective.com/help/contributing/development/api that your API provides some interface to **get** the information from opencollective.
We (the educode asbl) use your project and we would need some API to **insert information** such as 
- new events with their characteristics
  - date & time, possibly recurring and defined in a serie with a suite that may *not* be regular (each week at the same time) but irregular (one Monday, then the next Tuesday at a different time). A possible way to do that is to define a ID for the event serie
  - the different prices
  - the text to publish
  - in short : every characteristics of the events
Would you have any idea of the time frame such API could be developed and what the effort would be ? (we could students we are working with to help with the developpement)
These API are necessary to use your tool to be the solution to get the payment and registration of participants and be able to automatically insert our event from our own platform.
Best regards

**Comment by znarf** [comment]
@npettiaux I'm sorry, I'm only seeing your issue now because it was originally posted in the documentation repository.

We'll make Event creation available in our API soon, maybe some characteristics described in your issue will be missing though.

**Comment by znarf** [comment]
Duplicate with https://github.com/opencollective/opencollective/issues/5190
